src/cbmc/cbmc_parse_options.cpp

```diff
@@ -669,11 +669,12 @@ int cbmc_parse_optionst::doit()
     }
   }
 
-  // fall back until everything has been ported to goto-checker
+  // We should know now which verifier to execute.
   if(verifier == nullptr)
   {
-    return bmct::do_language_agnostic_bmc(
-      options, goto_model, ui_message_handler);
+    error() << "Given combination of command line arguments not supported"
+            << eom;
+    exit(CPROVER_EXIT_USAGE_ERROR);
```

Comment:
Is this even possible? The branching structure above makes it a bit hard to see, actually. Could `else if` please be used above to reduce indent? If it is genuinely possible, the error message should say which command line options are at fault for that.

Reply:
Alternative in 1b72e91.

Reply:
Can I ask for one level of indent by doing `if(options.get_bool_option("localize-faults")) {...} else {...}` to make the top level only decide on stop-on-fail+paths, in each of the (then) three cases?

Reply:
See f157b2b

Reply:
Nice! Which now makes clear that the final `else` is dead code, because you've enumerated all four combinations of paths/stop-on-fail.